### DISCUSSION TITLE
MacOS won't accept password

### DISCUSSION THREAD

**Discussion by crippit** [mbentley/docker-timemachine#187] [Help]
Hi, the password that is set in the docker compose file (in my case TIME) isn't accepted by MacOS, it just jiggles when entering the password as if it is wrong.  It can see the drive, just not authenticate against it. Ideas?

**Comment by mbentley** [comment]
Could you share your compose file and container logs?

**Comment by crippit** [comment]
Here is docker compose:

```
version: "3.7"
services:
  timebackup:
    network_mode: "host"
    environment:
      - TM_USERNAME=TIME
      - TM_GROUPNAME=TIME
      - PASSWORD=TIME
      - TM_UID=1000
      - TM_GID=1000
      - SET_PERMISSIONS=false
      - VOLUME_SIZE_LIMIT=0
    restart: unless-stopped
    ports:
      - "137:137/udp"
      - "138:138/udp"
      - "139:139"
      - "445:445"
    volumes:
      - /media/chris/macbackup1/timemachine:/opt/TIME \
    tmpfs:
      - /run/samba
    ulimits:
      nofile:
        soft: 65536
        hard: 65536
    container_name: timemachine
    image: mbentley/timemachine:smb
```

and here are the container logs

```
INFO: CUSTOM_SMB_CONF=false; generating [global] section of /etc/samba/smb.conf...
INFO: Creating /var/log/samba/cores
INFO: Avahi - generating base configuration in /etc/avahi/services/smbd.service...
INFO: Avahi - adding the 'dk0', 'TimeMachine' share txt-record to /etc/avahi/services/smbd.service...
INFO: Group TIME doesn't exist; creating...
INFO: User TIME doesn't exist; creating...
INFO: Setting password from environment variable
chpasswd: password for 'TIME' changed
INFO: INFO: CUSTOM_SMB_CONF=false; generating [TimeMachine] section of /etc/samba/smb.conf...
INFO: Samba - Created Added user TIME.
INFO: Samba - Enabled user TIME.
INFO: Samba - setting password
INFO: SET_PERMISSIONS=false; not setting ownership and permissions for /opt/TIME
INFO: Avahi - completing the configuration in /etc/avahi/services/smbd.service...
INFO: running test for xattr support on your time machine persistent storage location...
INFO: xattr test successful - your persistent data store supports xattrs
WARN: Detected filesystem for /opt/TIME is overlay! This likely means that your data is being stored inside the container, not in a volume! See https://github.com/mbentley/docker-timemachine#persistent-data-path
INFO: Detected filesystem for /opt/TIME \ is ext4
INFO: entrypoint complete; executing 's6-svscan /etc/s6'
dbus socket not yet available; sleeping...
nmbd version 4.19.6 started.
Copyright Andrew Tridgell and the Samba Team 1992-2023
smbd version 4.19.6 started.
Copyright Andrew Tridgell and the Samba Team 1992-2023
Found user 'avahi' (UID 86) and group 'avahi' (GID 86).
Successfully dropped root privileges.
avahi-daemon 0.8 starting up.
WARNING: No NSS support for mDNS detected, consider installing nss-mdns!
Loading service file /etc/avahi/services/smbd.service.
*** WARNING: Detected another IPv4 mDNS stack running on this host. This makes mDNS unreliable and is thus not recommended. ***
*** WARNING: Detected another IPv6 mDNS stack running on this host. This makes mDNS unreliable and is thus not recommended. ***
Joining mDNS multicast group on interface veth48849c1.IPv6 with address fe80::c02:83ff:fe21:b025.
New relevant interface veth48849c1.IPv6 for mDNS.
Joining mDNS multicast group on interface veth2309880.IPv6 with address fe80::cb1:e6ff:fe26:876a.
New relevant interface veth2309880.IPv6 for mDNS.
Joining mDNS multicast group on interface vethb2a6683.IPv6 with address fe80::8887:5cff:fe1e:4676.
New relevant interface vethb2a6683.IPv6 for mDNS.
Joining mDNS multicast group on interface veth2ac098a.IPv6 with address fe80::6494:eff:fe28:7007.
New relevant interface veth2ac098a.IPv6 for mDNS.
Joining mDNS multicast group on interface veth986d18a.IPv6 with address fe80::7c99:d7ff:feeb:8164.
New relevant interface veth986d18a.IPv6 for mDNS.
Joining mDNS multicast group on interface veth460e9fd.IPv6 with address fe80::b45c:92ff:fee5:d801.
New relevant interface veth460e9fd.IPv6 for mDNS.
Joining mDNS multicast group on interface vethbbd55a8.IPv6 with address fe80::bc82:f6ff:fe2b:cd5a.
New relevant interface vethbbd55a8.IPv6 for mDNS.
Joining mDNS multicast group on interface veth2df3ca3.IPv6 with address fe80::b893:9ff:fe8f:9f1f.
New relevant interface veth2df3ca3.IPv6 for mDNS.
Joining mDNS multicast group on interface vethd957c21.IPv6 with address fe80::9081:7ff:fe9c:722d.
New relevant interface vethd957c21.IPv6 for mDNS.
Joining mDNS multicast group on interface vethb195b84.IPv6 with address fe80::fc2f:40ff:fec4:5c47.
New relevant interface vethb195b84.IPv6 for mDNS.
Joining mDNS multicast group on interface vethaafd13d.IPv6 with address fe80::b05d:eff:fef8:afb4.
New relevant interface vethaafd13d.IPv6 for mDNS.
Joining mDNS multicast group on interface veth353ba08.IPv6 with address fe80::745b:ccff:fe15:f11e.
New relevant interface veth353ba08.IPv6 for mDNS.
Joining mDNS multicast group on interface veth41c557b.IPv6 with address fe80::ac9c:89ff:fea7:5791.
New relevant interface veth41c557b.IPv6 for mDNS.
Joining mDNS multicast group on interface veth30362b1.IPv6 with address fe80::cccb:9ff:fe16:3564.
New relevant interface veth30362b1.IPv6 for mDNS.
Joining mDNS multicast group on interface vetha94f5dc.IPv6 with address fe80::306a:60ff:fe35:1aa9.
New relevant interface vetha94f5dc.IPv6 for mDNS.
Joining mDNS multicast group on interface veth7ccd5f9.IPv6 with address fe80::bc11:c4ff:fefd:2938.
New relevant interface veth7ccd5f9.IPv6 for mDNS.
Joining mDNS multicast group on interface veth1a3dddd.IPv6 with address fe80::fc62:9ff:fed0:fe91.
New relevant interface veth1a3dddd.IPv6 for mDNS.
Joining mDNS multicast group on interface veth2aedf14.IPv6 with address fe80::640d:d5ff:fe3d:5bf0.
New relevant interface veth2aedf14.IPv6 for mDNS.
Joining mDNS multicast group on interface vethc238bc3.IPv6 with address fe80::d0ba:adff:fe7c:4008.
New relevant interface vethc238bc3.IPv6 for mDNS.
Joining mDNS multicast group on interface veth4555aba.IPv6 with address fe80::b828:42ff:fea3:17e0.
New relevant interface veth4555aba.IPv6 for mDNS.
Joining mDNS multicast group on interface vethc249a2c.IPv6 with address fe80::4c49:b5ff:fefe:ab44.
New relevant interface vethc249a2c.IPv6 for mDNS.
Joining mDNS multicast group on interface vethdbed149.IPv6 with address fe80::49d:c0ff:fe80:1fdb.
New relevant interface vethdbed149.IPv6 for mDNS.
Joining mDNS multicast group on interface hassio.IPv6 with address fe80::42:2eff:fee4:6582.
New relevant interface hassio.IPv6 for mDNS.
Joining mDNS multicast group on interface hassio.IPv4 with address 172.30.32.1.
New relevant interface hassio.IPv4 for mDNS.
Joining mDNS multicast group on interface docker0.IPv6 with address fe80::42:4dff:fe48:3358.
New relevant interface docker0.IPv6 for mDNS.
Joining mDNS multicast group on interface docker0.IPv4 with address 172.17.0.1.
New relevant interface docker0.IPv4 for mDNS.
Joining mDNS multicast group on interface enp0s25.IPv4 with address 192.168.0.18.
New relevant interface enp0s25.IPv4 for mDNS.
Joining mDNS multicast group on interface lo.IPv6 with address ::1.
New relevant interface lo.IPv6 for mDNS.
Joining mDNS multicast group on interface lo.IPv4 with address 127.0.0.1.
New relevant interface lo.IPv4 for mDNS.
Network interface enumeration completed.
Registering new address record for fe80::c02:83ff:fe21:b025 on veth48849c1.*.
Registering new address record for fe80::cb1:e6ff:fe26:876a on veth2309880.*.
Registering new address record for fe80::8887:5cff:fe1e:4676 on vethb2a6683.*.
Registering new address record for fe80::6494:eff:fe28:7007 on veth2ac098a.*.
Registering new address record for fe80::7c99:d7ff:feeb:8164 on veth986d18a.*.
Registering new address record for fe80::b45c:92ff:fee5:d801 on veth460e9fd.*.
Registering new address record for fe80::bc82:f6ff:fe2b:cd5a on vethbbd55a8.*.
Registering new address record for fe80::b893:9ff:fe8f:9f1f on veth2df3ca3.*.
Registering new address record for fe80::9081:7ff:fe9c:722d on vethd957c21.*.
Registering new address record for fe80::fc2f:40ff:fec4:5c47 on vethb195b84.*.
Registering new address record for fe80::b05d:eff:fef8:afb4 on vethaafd13d.*.
Registering new address record for fe80::745b:ccff:fe15:f11e on veth353ba08.*.
Registering new address record for fe80::ac9c:89ff:fea7:5791 on veth41c557b.*.
Registering new address record for fe80::cccb:9ff:fe16:3564 on veth30362b1.*.
Registering new address record for fe80::306a:60ff:fe35:1aa9 on vetha94f5dc.*.
Registering new address record for fe80::bc11:c4ff:fefd:2938 on veth7ccd5f9.*.
Registering new address record for fe80::fc62:9ff:fed0:fe91 on veth1a3dddd.*.
Registering new address record for fe80::640d:d5ff:fe3d:5bf0 on veth2aedf14.*.
Registering new address record for fe80::d0ba:adff:fe7c:4008 on vethc238bc3.*.
Registering new address record for fe80::b828:42ff:fea3:17e0 on veth4555aba.*.
Registering new address record for fe80::4c49:b5ff:fefe:ab44 on vethc249a2c.*.
Registering new address record for fe80::49d:c0ff:fe80:1fdb on vethdbed149.*.
Registering new address record for fe80::42:2eff:fee4:6582 on hassio.*.
Registering new address record for 172.30.32.1 on hassio.IPv4.
Registering new address record for fe80::42:4dff:fe48:3358 on docker0.*.
Registering new address record for 172.17.0.1 on docker0.IPv4.
Registering new address record for 192.168.0.18 on enp0s25.IPv4.
Registering new address record for ::1 on lo.*.
Registering new address record for 127.0.0.1 on lo.IPv4.
Server startup complete. Host name is Server.local. Local service cookie is 4075642582.
Service "Server" (/etc/avahi/services/smbd.service) successfully established.
become_local_master_stage2: *****

Samba name server SERVER is now a local master browser for workgroup WORKGROUP on subnet 172.17.0.1

*****
become_local_master_stage2: *****

Samba name server SERVER is now a local master browser for workgroup WORKGROUP on subnet 172.30.32.1

*****
become_local_master_stage2: *****

Samba name server SERVER is now a local master browser for workgroup WORKGROUP on subnet 192.168.0.18

*****
```


Thank you!

**Reply by mbentley** [reply]
Your volume line has an issue:

Change this:
`   - /media/chris/macbackup1/timemachine:/opt/TIME \`
to this:
`   - /media/chris/macbackup1/timemachine:/opt/TIME`

(Remove the trailing `\`)

These lines in the logs confirm that this wasn't just a copy and paste error in this case:

```
WARN: Detected filesystem for /opt/TIME is overlay! This likely means that your data is being stored inside the container, not in a volume! See https://github.com/mbentley/docker-timemachine#persistent-data-path
INFO: Detected filesystem for /opt/TIME \ is ext4
```

**Reply by crippit** [reply]
This is still jiggling after changing this. Here are my latest docker logs

> `INFO: CUSTOM_SMB_CONF=false; generating [global] section of /etc/samba/smb.conf...
> INFO: Creating /var/log/samba/cores
> INFO: Avahi - generating base configuration in /etc/avahi/services/smbd.service...
> INFO: Avahi - adding the 'dk0', 'TimeMachine' share txt-record to /etc/avahi/services/smbd.service...
> INFO: Group TIME doesn't exist; creating...
> INFO: User TIME doesn't exist; creating...
> INFO: Setting password from environment variable
> chpasswd: password for 'TIME' changed
> INFO: INFO: CUSTOM_SMB_CONF=false; generating [TimeMachine] section of /etc/samba/smb.conf...
> INFO: Samba - Created Added user TIME.
> INFO: Samba - Enabled user TIME.
> INFO: Samba - setting password
> INFO: SET_PERMISSIONS=false; not setting ownership and permissions for /opt/TIME
> INFO: Avahi - completing the configuration in /etc/avahi/services/smbd.service...
> INFO: running test for xattr support on your time machine persistent storage location...
> INFO: xattr test successful - your persistent data store supports xattrs
> INFO: Detected filesystem for /opt/TIME is ext4
> INFO: entrypoint complete; executing 's6-svscan /etc/s6'
> dbus socket not yet available; sleeping...
> nmbd version 4.19.6 started.
> Copyright Andrew Tridgell and the Samba Team 1992-2023
> smbd version 4.19.6 started.
> Copyright Andrew Tridgell and the Samba Team 1992-2023
> Found user 'avahi' (UID 86) and group 'avahi' (GID 86).
> Successfully dropped root privileges.
> avahi-daemon 0.8 starting up.
> WARNING: No NSS support for mDNS detected, consider installing nss-mdns!
> Loading service file /etc/avahi/services/smbd.service.
> *** WARNING: Detected another IPv4 mDNS stack running on this host. This makes mDNS unreliable and is thus not recommended. ***
> *** WARNING: Detected another IPv6 mDNS stack running on this host. This makes mDNS unreliable and is thus not recommended. ***
> Joining mDNS multicast group on interface vethbf178bf.IPv6 with address fe80::3411:bcff:fe2f:6a63.
> New relevant interface vethbf178bf.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth48849c1.IPv6 with address fe80::c02:83ff:fe21:b025.
> New relevant interface veth48849c1.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth2309880.IPv6 with address fe80::cb1:e6ff:fe26:876a.
> New relevant interface veth2309880.IPv6 for mDNS.
> Joining mDNS multicast group on interface vethb2a6683.IPv6 with address fe80::8887:5cff:fe1e:4676.
> New relevant interface vethb2a6683.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth2ac098a.IPv6 with address fe80::6494:eff:fe28:7007.
> New relevant interface veth2ac098a.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth986d18a.IPv6 with address fe80::7c99:d7ff:feeb:8164.
> New relevant interface veth986d18a.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth460e9fd.IPv6 with address fe80::b45c:92ff:fee5:d801.
> New relevant interface veth460e9fd.IPv6 for mDNS.
> Joining mDNS multicast group on interface vethbbd55a8.IPv6 with address fe80::bc82:f6ff:fe2b:cd5a.
> New relevant interface vethbbd55a8.IPv6 for mDNS.
> Joining mDNS multicast group on interface vethd957c21.IPv6 with address fe80::9081:7ff:fe9c:722d.
> New relevant interface vethd957c21.IPv6 for mDNS.
> Joining mDNS multicast group on interface vethb195b84.IPv6 with address fe80::fc2f:40ff:fec4:5c47.
> New relevant interface vethb195b84.IPv6 for mDNS.
> Joining mDNS multicast group on interface vethaafd13d.IPv6 with address fe80::b05d:eff:fef8:afb4.
> New relevant interface vethaafd13d.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth353ba08.IPv6 with address fe80::745b:ccff:fe15:f11e.
> New relevant interface veth353ba08.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth41c557b.IPv6 with address fe80::ac9c:89ff:fea7:5791.
> New relevant interface veth41c557b.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth30362b1.IPv6 with address fe80::cccb:9ff:fe16:3564.
> New relevant interface veth30362b1.IPv6 for mDNS.
> Joining mDNS multicast group on interface vetha94f5dc.IPv6 with address fe80::306a:60ff:fe35:1aa9.
> New relevant interface vetha94f5dc.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth7ccd5f9.IPv6 with address fe80::bc11:c4ff:fefd:2938.
> New relevant interface veth7ccd5f9.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth1a3dddd.IPv6 with address fe80::fc62:9ff:fed0:fe91.
> New relevant interface veth1a3dddd.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth2aedf14.IPv6 with address fe80::640d:d5ff:fe3d:5bf0.
> New relevant interface veth2aedf14.IPv6 for mDNS.
> Joining mDNS multicast group on interface vethc238bc3.IPv6 with address fe80::d0ba:adff:fe7c:4008.
> New relevant interface vethc238bc3.IPv6 for mDNS.
> Joining mDNS multicast group on interface veth4555aba.IPv6 with address fe80::b828:42ff:fea3:17e0.
> New relevant interface veth4555aba.IPv6 for mDNS.
> Joining mDNS multicast group on interface vethc249a2c.IPv6 with address fe80::4c49:b5ff:fefe:ab44.
> New relevant interface vethc249a2c.IPv6 for mDNS.
> Joining mDNS multicast group on interface vethdbed149.IPv6 with address fe80::49d:c0ff:fe80:1fdb.
> New relevant interface vethdbed149.IPv6 for mDNS.
> Joining mDNS multicast group on interface hassio.IPv6 with address fe80::42:2eff:fee4:6582.
> New relevant interface hassio.IPv6 for mDNS.
> Joining mDNS multicast group on interface hassio.IPv4 with address 172.30.32.1.
> New relevant interface hassio.IPv4 for mDNS.
> Joining mDNS multicast group on interface docker0.IPv6 with address fe80::42:4dff:fe48:3358.
> New relevant interface docker0.IPv6 for mDNS.
> Joining mDNS multicast group on interface docker0.IPv4 with address 172.17.0.1.
> New relevant interface docker0.IPv4 for mDNS.
> Joining mDNS multicast group on interface enp0s25.IPv4 with address 192.168.0.18.
> New relevant interface enp0s25.IPv4 for mDNS.
> Joining mDNS multicast group on interface lo.IPv6 with address ::1.
> New relevant interface lo.IPv6 for mDNS.
> Joining mDNS multicast group on interface lo.IPv4 with address 127.0.0.1.
> New relevant interface lo.IPv4 for mDNS.
> Network interface enumeration completed.
> Registering new address record for fe80::3411:bcff:fe2f:6a63 on vethbf178bf.*.
> Registering new address record for fe80::c02:83ff:fe21:b025 on veth48849c1.*.
> Registering new address record for fe80::cb1:e6ff:fe26:876a on veth2309880.*.
> Registering new address record for fe80::8887:5cff:fe1e:4676 on vethb2a6683.*.
> Registering new address record for fe80::6494:eff:fe28:7007 on veth2ac098a.*.
> Registering new address record for fe80::7c99:d7ff:feeb:8164 on veth986d18a.*.
> Registering new address record for fe80::b45c:92ff:fee5:d801 on veth460e9fd.*.
> Registering new address record for fe80::bc82:f6ff:fe2b:cd5a on vethbbd55a8.*.
> Registering new address record for fe80::9081:7ff:fe9c:722d on vethd957c21.*.
> Registering new address record for fe80::fc2f:40ff:fec4:5c47 on vethb195b84.*.
> Registering new address record for fe80::b05d:eff:fef8:afb4 on vethaafd13d.*.
> Registering new address record for fe80::745b:ccff:fe15:f11e on veth353ba08.*.
> Registering new address record for fe80::ac9c:89ff:fea7:5791 on veth41c557b.*.
> Registering new address record for fe80::cccb:9ff:fe16:3564 on veth30362b1.*.
> Registering new address record for fe80::306a:60ff:fe35:1aa9 on vetha94f5dc.*.
> Registering new address record for fe80::bc11:c4ff:fefd:2938 on veth7ccd5f9.*.
> Registering new address record for fe80::fc62:9ff:fed0:fe91 on veth1a3dddd.*.
> Registering new address record for fe80::640d:d5ff:fe3d:5bf0 on veth2aedf14.*.
> Registering new address record for fe80::d0ba:adff:fe7c:4008 on vethc238bc3.*.
> Registering new address record for fe80::b828:42ff:fea3:17e0 on veth4555aba.*.
> Registering new address record for fe80::4c49:b5ff:fefe:ab44 on vethc249a2c.*.
> Registering new address record for fe80::49d:c0ff:fe80:1fdb on vethdbed149.*.
> Registering new address record for fe80::42:2eff:fee4:6582 on hassio.*.
> Registering new address record for 172.30.32.1 on hassio.IPv4.
> Registering new address record for fe80::42:4dff:fe48:3358 on docker0.*.
> Registering new address record for 172.17.0.1 on docker0.IPv4.
> Registering new address record for 192.168.0.18 on enp0s25.IPv4.
> Registering new address record for ::1 on lo.*.
> Registering new address record for 127.0.0.1 on lo.IPv4.
> Server startup complete. Host name is Server.local. Local service cookie is 332434068.
> Service "Server" (/etc/avahi/services/smbd.service) successfully established.
> become_local_master_stage2: *****
> 
> Samba name server SERVER is now a local master browser for workgroup WORKGROUP on subnet 172.17.0.1
> 
> *****
> become_local_master_stage2: *****
> 
> Samba name server SERVER is now a local master browser for workgroup WORKGROUP on subnet 172.30.32.1
> 
> *****`

**Reply by crippit** [reply]
Also, if it helps, here are my timemachine logs:

> `2024-11-04 15:23:51  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:01  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:08  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:08  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:51  Couldn't find en0. Using hostUUIDDerivedMachineAddress...
> 2024-11-04 15:24:53  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:53  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:53  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:53  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:53  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:53  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:53  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:53  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:24:54  Running for notifyd event com.apple.system.powersources.source
> 2024-11-04 15:24:59  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:25:00  Running for notifyd event com.apple.mobile.keybagd.lock_status
> 2024-11-04 15:25:00  TMPowerState: 2
> 2024-11-04 15:25:00  Triggered for screen unlock!
> 2024-11-04 15:25:00  Running for notifyd event com.apple.mobile.keybagd.lock_status
> 2024-11-04 15:25:00  TMPowerState: 2
> 2024-11-04 15:25:00  Triggered for screen unlock!
> 2024-11-04 15:25:45  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:25:45  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:25:46  No pending apple event to handle
> 2024-11-04 15:25:47  Bonjour took 0.2 seconds to resolve 'smb://Server._smb._tcp.local./TimeMachine' to 'smb://Server.local/TimeMachine'
> 2024-11-04 15:25:47  Failed to create volume info from disk '<TMDisk: 0x600001c0a430> '/System/Volumes/Data/home'', error: missingName
> 2024-11-04 15:25:48  Choosing destination volume
> 2024-11-04 15:25:49  Prepare destination
> 2024-11-04 15:25:49  NetAuth session for 'smb://Server._smb._tcp.local./TimeMachine' will use options: {
>     AuthType = Time     AuthType = TimeMachine;
> 2024-11-04 15:25:49  Reporting preparation progress message: 'Mounting “TimeMachine” from “Server.local”…', fractionDone: nil
> 2024-11-04 15:25:57  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:26:01  Failed to create volume info from disk '<TMDisk: 0x600001c447b0> '/System/Volumes/Data/home'', error: missingName
> 2024-11-04 15:26:04  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:26:48  Failed to create volume info from disk '<TMDisk: 0x600001c55d40> '/System/Volumes/Data/home'', error: missingName
> 2024-11-04 15:27:04  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:27:19  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:27:19  com.apple.backupd.sandbox.xpc: connection invalid
> 2024-11-04 15:27:49  Failed to create volume info from disk '<TMDisk: 0x600001c79020> '/System/Volumes/Data/home'', error: missingName
> 2024-11-04 15:27:54  NAOpenSessionAsync reports sessionStatus: -128, sessionInfo: (null)
> 2024-11-04 15:27:54  User cancelled connection attempt (from NetAuth open dialog) for 'smb://Server._smb._tcp.local./TimeMachine'
> 2024-11-04 15:27:54  Choosing destination volume`
